cardano-cli/src/Cardano/CLI/Commands/Governance.hs

```diff
@@ -31,41 +32,34 @@ runGovernanceCreateVoteCmd
   -> ExceptT GovernanceCmdError IO ()
 runGovernanceCreateVoteCmd anyEra vChoice vType (govActionTxId, govActionIndex) votingStakeCred oFp = do
```

Comment:
We should deprecate this command as we are maintaining two vote creation commands and this one doesn't allow vote creation for DReps or committee members.